app/presenters/formats/edition_format_presenter.rb

```diff
@@ -49,6 +49,13 @@ def schema_name
       "override me"
     end
 
+    def document_type
+      # alternatively this could just be schema_name since schema and
```

Comment:
Given that we expect `document_type` and `schema` to be the same, shouldn't we define them in the same way?  It feels like a bit of mental baggage to have to work out whether `edition.class.to_s.underscore.gsub('_edition', '')` is the same as `"answer"`.  Am happy to go either way on how to do it.